localisation/00_zuwc_united_localisation.csv

```diff
@@ -2430,7 +2419,7 @@ EVTDESC_WCGOV_7992;Our duodecennial held elections are coming up a year from now
 EVTOPTC_WCTHW_2036;Enough! We are returning home.;;Genug! Wir kehren nach Hause zur�ck.;;;;;;;;;;;x
 nick_the_outcast;the Outcast;[From.GetLeLa] Banni[From.Get_E];[From.Getderdie] Ausgesto�ene;;;;;;;;;;;x
 faction_government;Political Faction;Faction politique;Politische Fraktion;;;;;;;;;;;x
-faction_government_desc;A union of many kingdoms and tribes. The power of the supreme leader is severely limited in order to provide a guarantee of self-government to the rulers within the political faction.;Une faction politique unifie de nombreux royaumes et de nombreuses tribus. Le pouvoir du chef supr�me est s�v�rement limit� afin de fournir aux dirigeants de la faction une garantie d'autonomie.;Eine Vereinigung vieler K�nigreiche und St�mme. Die Macht des obersten F�hrers ist stark eingeschr�nkt, um den Herrschern innerhalb der politischen Fraktion eine Garantie der Selbstverwaltung zu geben.;;;;;;;;;;;x
+faction_government_desc;A union of many Kingdoms and Tribes. The power of the supreme leader is severely limited in order to provide a guarantee of self-government to the rulers within the political faction.;Une faction politique unifie de nombreux royaumes et de nombreuses tribus. Le pouvoir du chef supr�me est s�v�rement limit� afin de fournir aux dirigeants de la faction une garantie d'autonomie.;Eine Vereinigung vieler K�nigreiche und St�mme. Die Macht des obersten F�hrers ist stark eingeschr�nkt, um den Herrschern innerhalb der politischen Fraktion eine Garantie der Selbstverwaltung zu geben.;;;;;;;;;;;x
```

Comment:
Why `Kingdoms and Tribes` are capitalized?

Reply:
reverted